### ISSUE TITLE
[Feature Request] Swipe between tabs

### ISSUE THREAD

**Issue by forkbomb9** [howeyc/crocgui#2]
It would be nice if the users could swipe between the tabs, it's rather intuitive on Android.

I have absolutely no experience with Go, so I don't know if it's even feasible.

**Comment by howeyc** [comment]
Yeah, there has been a discussion about that recently in Fyne slack (Fyne is the go graphics library I use). There was mention of a way it could be implemented.

I might give it a try eventually.

**Comment by forkbomb9** [comment]
Wow, that was the fastest reply I've ever seen in GH!
I may dig in the codebase and learn another language. If I can help in any way (testing, feedback...) just tell me.
